bloom.c

```diff
@@ -45,12 +45,13 @@ static uint32_t seed_murmur3(uint32_t seed, const char *data, int len)
 
 	int len4 = len / sizeof(uint32_t);
 
-	const uint32_t *blocks = (const uint32_t*)data;
-
 	uint32_t k;
-	for (i = 0; i < len4; i++)
-	{
-		k = blocks[i];
+	for (i = 0; i < len4; i++) {	
```

Comment:
the comment says this commit is an uint64 to uint8 on processing the data, but this section is/was dealing with uint32 data.  not sure if this change goes in this commit.

also, would this be a place to use get_be32() instead of the byte[1234] vars?  not sure, but curious.  (on my ipad here, so it’s a little hard to double check.)